third_party/proto/README.md

```diff
@@ -2,6 +2,14 @@
 
 To generate the protos that were present in this folder run:
 
+## v0.47.x+
+
+```bash
+buf export buf.build/cosmos/cosmos-sdk --output .
+```
+
+## v0.46.x
+
 ```bash
-buf export buf.build/cosmos/cosmos-sdk:$(curl -sS https://api.github.com/repos/cosmos/cosmos-sdk/commits/<sdk_version_tag> | jq -r .sha) --output .
+buf export buf.build/cosmos/cosmos-sdk:8cb30a2c4de74dc9bd8d260b1e75e176 --output .
```

Comment:
After some thought, I think we should maintain a table like this instead of pushing at any tag.